CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# CONTRIBUTING
+
+Welcome to CONTRIBUTING.md. Since you find this page, you might want to make a contribution to our [tongji-undergrad-thesis](https://github.com/TJ-CSCCG/tongji-undergrad-thesis)! Let's read these following parts and start contributing right now!
+
```

Comment:
Can we be brief? by deleting this sentence.

Reply:
I feel sorry I did such self-touching thing again. I'll review all my *.md files committed in this PR and remove all thsi kind of sentences.